packages/mobile/src/components/challenge-rewards-drawer/ProfileCompletionChecks.tsx

```diff
@@ -71,7 +71,7 @@ export const ProfileCompletionChecks = ({
   const goToProfile = useCallback(() => {
     onClose()
     if (currentUser?.handle) {
-      navigation.goBack()
+      navigation.navigate('Profile', { handle: currentUser?.handle })
```

Comment:
nit: since the if statement has currentUser?.handle, you can remove the ? from inside the if block. 
`navigation.navigate('Profile', { handle: currentUser.handle })`